frontend/packages/console-dynamic-plugin-sdk/src/shared-modules-init.ts

```diff
@@ -10,6 +10,7 @@ const modules: SharedModuleResolution = {
     require('@console/dynamic-plugin-sdk/src/lib-internal'),
   '@patternfly/react-core': async () => () => require('@patternfly/react-core'),
   '@patternfly/react-table': async () => () => require('@patternfly/react-table'),
+  '@patternfly/quickstarts': async () => () => require('@patternfly/quickstarts'),
```

Comment:
@spadgett @vojtechszocs just want to confirm you're okay with `@patternfly/quickstarts` as shared module